<launch>
	<arg name="quad" default="SQ01s"/>
	<arg name="world_name" default="$(find acl_sim)/worlds/density01seed0.world"/>
	
	<!-- We resume the logic in empty_world.launch, changing only the name of the world to be launched -->
	<include file="$(find gazebo_ros)/launch/empty_world.launch">
		<arg name="world_name" value="$(arg world_name)"/>
	</include>


</launch>